packages/sfui/frameworks/nuxt/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@storefront-ui/nuxt",
+  "version": "1.0.0",
+  "homepage": "https://docs.storefrontui.io/v2/",
+  "installConfig": {
+    "hoistingLimits": "workspaces"
```

Comment:
I guess that's needed? :(

Reply:
yeah - without this, there were even more problems with `eslint` and some other dependencies coming from `nuxt/kit`